archive-locations.php

```diff
@@ -4,25 +4,101 @@
 <?php dt_print_breadcrumbs( null, __( "Locations" ) ); ?>
 
 <div id="content">
-
-    <div id="inner-content" class="row">
-
-        <main id="main" class="large-8 medium-8 columns" role="main">
-
-            <?php get_template_part( 'parts/content', 'locations-tabs.php' ); ?>
-
+    
+    <div id="inner-content" class="grid-x grid-margin-x">
+        
+        <main id="main" class="large-8 medium-8 cell" role="main">
+            
+            <?php
+            $args = array(
+                'post_type' => 'locations',
+            
+            );
+            $query1 = new WP_Query( $args );
+            ?>
+            <?php if ( $query1->have_posts() ) : while ( $query1->have_posts() ) : $query1->the_post(); ?>
+                
+                <!-- To see additional archive styles, visit the /parts directory -->
+                <?php get_template_part( 'parts/loop', 'prayer' ); ?>
+            
+            <?php endwhile; ?>
+                
+                <?php disciple_tools_page_navi(); ?>
+            
+            <?php else : ?>
+                
+                <?php get_template_part( 'parts/content', 'missing' ); ?>
+            
+            <?php endif; ?>
+        
         </main> <!-- end #main -->
-
-        <aside class="large-4 medium-4 columns ">
-
+        
+        <aside class="large-4 medium-4 cell">
+            
             <section class="bordered-box">
-
-                <p>Links</p>
-
+                
+                <?php include 'searchform.php'; ?>
+            
             </section>
-
+            
+            <section class="bordered-box">
+                
+                <h4>Recent Posts</h4>
+                
+                <?php $args = array(
+                    'numberposts' => 10,
+                    'offset' => 0,
+                    'category' => 0,
+                    'orderby' => 'post_date',
+                    'order' => 'DESC',
+                    'include' => '',
+                    'exclude' => '',
+                    'meta_key' => '',
+                    'meta_value' =>'',
+                    'post_type' => 'locations',
+                    'post_status' => 'draft, publish, future, pending, private',
+                    'suppress_filters' => true
+                );
+                
+                $recent_posts = wp_get_recent_posts( $args, ARRAY_A );
```

Comment:
What is the purpose of recent locations?
Locations are probably set up all at the same time. So should all have the post_date